moveit_planners/pilz_industrial_motion_planner/src/trajectory_functions.cpp

```diff
@@ -40,7 +40,8 @@
 #include <tf2_kdl/tf2_kdl.h>
 #include <tf2_geometry_msgs/tf2_geometry_msgs.h>
 
-bool pilz_industrial_motion_planner::computePoseIK(const moveit::core::RobotModelConstPtr& robot_model,
+bool pilz_industrial_motion_planner::computePoseIK(const planning_scene::PlanningSceneConstPtr& scene,
+                                                   const moveit::core::RobotModelConstPtr& robot_model,
```

Comment:
Can't you get rid off the RobotModel argument? The PlanningScene is linked to a RobotModel.
This comment applies to several similar methods below!

Reply:
Sure thing :+1: